content/en/getting_started/synthetics/_index.md

```diff
@@ -0,0 +1,215 @@
+
+3. Click the **Save test** button. Your test is now populated in its own page, which includes details on properties, history, and test results.
+
+The **Synthetics Tests** page is useful for monitoring changes over time and verifying request anomalies across all of your synthetics tests. You can monitor this as well as events and test results to expedite troubleshooting during downtime. Once you create a few tests, preview all of tests in realtime on this page.
+
+### Create a browser test
+
+[Browser tests][9] are scenarios executed by Datadog on your web applications. They run at configurable periodic intervals from multiple locations around the world, and from multiple devices. These checks verify both that your applications are up and responding to requests, and that any conditions defined in your scenarios are met.
+
+To create a browser test:
+
+1. In the Datadog application, hover over **UX Monitoring** in the left hand menu and select *Synthetics Test*. 
+2. In the top right corner, click the *New Test* button. 
+3. Select *Browser Test*. 
+4. Define the configuration of your browser test.
+
+1. **Starting URL**: Set the URL from which your browser test starts the scenario.
+    * Advanced Options (optional): Set custom request headers, cookies, or authenticate through Basic Auth.
+        * Headers: Defined headers override the default browser headers. For example, set the User Agent in the header to [identify Datadog scripts][10].
+        * Authentication: Authenticate through HTTP Basic Authentication with a username and a password.
+        * Cookies: Defined cookies are added to the default browser cookies. Set multiple cookies using the format `<COOKIE_NAME1>=<COOKIE_VALUE1>; <COOKIE_NAME2>=<COOKIE_VALUE2>`.
+
+2. **Name**: Set the name of your browser test.
+3. **Select your tags**: Select the tags attached to your browser test. Use the `<KEY>:<VALUE>` format to filter on a `<VALUE>` for a given `<KEY>` on the Synthetics page.
+4. **Devices**: Select which devices to run your check on. Available devices are `Laptop Large`, `Tablet`, and `Mobile Small`.
+5. **Locations**: Select which Datadog managed locations to run the test from. Many AWS locations from around the world are available. The full list is retrievable through the [Datadog API][5]. You can also set up a [Private Location][11] to run a Synthetics Browser test on a private URL not accessible from the public internet.
+6. **How often should Datadog run the test?** Set the interval for how often the test will run. Intervals are available between every 15 minutes to once per week. [Contact support][12] to enable additional frequencies for your test.
+7. **Alert conditions**: Set the conditions that need to be met (`minutes`, `locations`, and `retry`) for a test to retry if it has failed.
```

Comment:
Set the conditions that need to be met (`minutes`, `locations`, and `retry`) to trigger an alert.